config/rbac/role.yaml

```diff
@@ -116,27 +117,9 @@ rules:
   resources:
   - telemetries
   verbs:
-  - create
-  - delete
   - get
```

Comment:
Do we need these to cluster scoped

Reply:
No, only get, list, and watch. Similar to other resources.